### PR TITLE
[Application] darknet53 nntrainer implementation for yolo v3 @open sesame 09/14 11:47

### PR DESCRIPTION
**Pull request by baek2sm**
Added nntrainer darknet53 model for yolo v3.

It's used in yolo v3 as a backbone model.

* ISSUE: Now it cannot import pre-trained weights
from pytorch model. It needs debugging for now.
I'll check and update it in later commit.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.com>

### CODE REVIEW

File: Applications/YOLOv3/PyTorch/yolo.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (myungjoo) [inline Applications/YOLOv3/PyTorch/yolo.py:1]: missing shebang?

Comment (baek2sm) [reply Applications/YOLOv3/PyTorch/yolo.py:2]: > missing shebang?

Thanks a lot! I added it. :)

**Comment by songgot** [comment]
please rebase #2177

**Comment by myungjoo** [comment]
@PTAL @DonghakPark @lhs8928 @jihochu @SeoHyungjun

**Review by DonghakPark — APPROVED** [review]
LGTM!
how about add prepare file for dataset

**Review by Hyoungsukim — APPROVED** [review]
lgtm

**Comment by baek2sm** [comment]
> how about add prepare file for dataset

I will upload some dataset without license issues later. Thanks.

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

**Comment by baek2sm** [comment]
It is already merged by #2229

**Pull request closed without merging**